documentation/client/docs/libraries/python/how_to/_07_create_block/_key_api_ref_links.md

```diff
@@ -1,7 +1,7 @@
-### `Client.build_and_post_block(secret_manager=None, options=None)`
+### `Client.build_and_post_block(secret_manager=None)`
 
 You can create a simple empty block by calling
-the [`Client.build_and_post_block(secret_manager=None, options=None)`](./../libraries/python/api_reference#build_and_post_block)
+the [`Client.build_and_post_block(secret_manager=None)`](./../libraries/python/api_reference#build_and_post_block)
```

Comment:
Should add the new params?

Reply:
I actually still don't know how this page is generated 😆. I think there I just need to use pydoc or something? @Thoralf-M do you know?

Reply:
Good that we have a readme where it's written :trollface: 
https://github.com/iotaledger/iota-sdk/blob/develop/sdk/src/client/bindings/python/README.md#generate-api-references

Reply:
I thought it was also autogenerated. But yeah will change manually then.
The API thingy is probably still completely broken anyway :trollface:

Reply:
Ok sorry, took me some time. But I had to figure out if this docs are even used right now, because I couldn't find them.
TL;DR: They aren't 😆 .

They were just an idea back then to reference to the API documentation. But Rust wasn't published yet. 🤔 

But anyway as this would/will(?) link to the actual function doc it should be fine to keep it simple